src/main/java/com/uid2/shared/store/parser/SiteParser.java

```diff
@@ -1,45 +1,24 @@
 package com.uid2.shared.store.parser;
 
-import com.uid2.shared.Utils;
-import com.uid2.shared.model.ClientType;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.uid2.shared.model.Site;
-import io.vertx.core.json.JsonArray;
-import io.vertx.core.json.JsonObject;
+import com.uid2.shared.util.Mapper;
 
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.HashMap;
-import java.util.HashSet;
+import java.util.Arrays;
 import java.util.Map;
-import java.util.Set;
 import java.util.stream.Collectors;
 
-public class SiteParser implements Parser<Map<Integer, Site>> {
+public class SiteParser implements Parser<Map<Integer,Site>> {
```

Comment:
Missing space after the , so it registered as a change

Reply:
oopsie, fixed